test-data/unit/check-incremental.test

```diff
@@ -3983,6 +3983,40 @@ main:2: error: Revealed type is 'def (x: builtins.list[builtins.int]) -> a.a.A'
 [out2]
 main:2: error: Revealed type is 'def (x: builtins.list[builtins.int]) -> a.a.A'
 
+[case testAttrsIncrementalConverterType]
+from a import C
+import attr
+o = C("1", "2", "3")
+o = C(1, 2, "3")
+reveal_type(C)
+@attr.s
+class D(C):
+    a: str = attr.ib()
+reveal_type(D)
+[file a.py]
+from typing import overload
+import attr
+class complex:
+    @overload
+    def __init__(self, re: float = 0.0, im: float = 0.0) -> None: ...
+    @overload
```

Comment:
Could you also add tests for an overloaded converter function?

Reply:
Thanks for this note.  It was actually broken!